.github/workflows/tests.yml

```diff
@@ -117,6 +117,7 @@ jobs:
     # comming from the FlexibleSUSY/FlexibleSUSY repo.
     # For non PR triggers we check for the name of the repo so that if someone
     # forks FS github doesn't try to run tests on their code.
+    if: false
```

Comment:
I guess you will remove this line before merging, right?

Reply:
Yes.

Reply:
Ok, thanks! Your PR is fine for me then.